snuba/consumers/strategy_factory.py

```diff
@@ -166,4 +169,7 @@ def flush_batch(
                 strategy, self.__max_messages_to_process, 10.0
             )
 
+        if self.__health_check_file is not None:
+            strategy = Healthcheck(self.__health_check_file, strategy)
```

Comment:
Just a question as I'm trying to understand this better, this means that the Healthcheck would be the first step of the ProcessingStrategy?

Reply:
yes